crates/dekaf/src/main.rs

```diff
@@ -195,8 +196,20 @@ async fn main() -> anyhow::Result<()> {
             let acceptor = acceptor.clone();
             tokio::select! {
                 accept = kafka_listener.accept() => {
-                    let (socket, addr) = accept?;
-                    let socket = acceptor.accept(socket).await?;
+                    let (socket, addr) = match accept {
+                        Ok(res) => res,
+                        Err(e) => {
+                            tracing::error!(e=e.to_string(), "Failed to accept TCP connection");
+                            continue;
+                        }
+                    };
+                    let socket = match acceptor.accept(socket).await {
+                        Ok(res) => res,
+                        Err(e) => {
+                            tracing::error!(e=e.to_string(), "Failed to accept TLS connection");
```

Comment:
IDK how common this has been, but I'll just put it out there that it could potentially result in a lot of log spam.

Reply:
Yeah, that's true... could conceivably be quite frequent if someone configures a connection using `SASL_PLAINTEXT` and forgets about it for a while. I think I'll just get rid of the logs